modules/social_features/social_group/social_group.api.php

```diff
@@ -79,7 +97,9 @@ function hook_social_group_overview_route_alter(array &$route, GroupInterface $g
  * @ingroup social_group_api
  */
 function hook_social_group_move(\Drupal\node\NodeInterface $node) {
```

Comment:
```suggestion
function hook_social_group_move(NodeInterface $node) {
```

Add a use statement at the top of the file instead.